src/Microsoft.Azure.SignalR.Common/Auth/AzureActiveDirectoryOptions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.SignalR
+{
+    public class AzureActiveDirectoryOptions
+    {
+        public readonly X509Certificate2 ClientCert;
+
+        public string ClientId { get; }
+
+        public string ClientSecret { get; }
+
+        public string TenantId { get; }
+
+        public bool Enabled { get; } = false;
+
+        public string Instance { get; set; } = "https://login.microsoftonline.com/{0}";
```

Comment:
What does `Instance` stands for? better naming?
Also please add triple-slash comments to public properties